dockstore-client/src/main/java/io/github/collaboratory/wdl/WDLClient.java

```diff
@@ -121,8 +124,8 @@ private File getCromwellTargetFile() {
      * @return an exit code for the run
      */
     @Override
-    public long launch(String entry, boolean isLocalEntry, String yamlRun, String jsonRun, String csvRuns, String wdlOutputTarget, String uuid)
-        throws ApiException {
+    @SuppressWarnings({"checkstyle:parameternumber", "checkstyle:methodlength"})
```

Comment:
There might be value in splitting out some of this into a new method rather than overriding the warning on method length.